package.json

```diff
@@ -349,8 +349,7 @@
         "compile": "tsc -p ./ && npm run lint && npm run buildScripts",
         "recompile": "npm run clean && npm run compile",
         "watch": "npm run buildScripts && tsc -watch -p ./",
-        "postinstall": "node ./node_modules/vscode/bin/install",
```

Comment:
Any reason why this was removed?

Reply:
See [guidance](https://code.visualstudio.com/api/working-with-extensions/testing-extension#migrating-from-vscode) - The reason is that the npm packages being used now do not require an explicit step to download the extension host -- this is handled within vscode-test.